rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  def initialize(book, person)
```

Comment:
- Please,  let's also add the `date` to the constructor so that the  `Book` and `Person` are set in it to further simplify the `many-to-many` relationship.

Reply:
same as last comment... it's automatically creating without the need to pass as paramenters